src/components/OSCALControlProse.js

```diff
@@ -359,21 +363,39 @@ export function OSCALReplacedProseWithParameterLabel(props) {
   }
 
   const { controlId, controlPartId } = props.modificationDisplay.props;
-  const statementIds = props.modificationDisplay.props.modificationAlters
+  const removeByIds = props.modificationDisplay.props.modificationAlters
     ?.find((item) => item["control-id"] === controlId)
     ?.removes?.find((object) => object["by-id"] === controlPartId);
-
-  if (!statementIds?.length) {
+  const removeByNames = props.modificationDisplay.props.modificationAlters
+    ?.find((item) => item["control-id"] === controlId)
+    ?.removes?.find((object) => object["by-name"] === controlPartId);
+  const removeByNS = props.modificationDisplay.props.modificationAlters
+    ?.find((item) => item["control-id"] === controlId)
+    ?.removes?.find((object) => object["by-ns"] === controlPartId);
+  const removeByClass = props.modificationDisplay.props.modificationAlters
+    ?.find((item) => item["control-id"] === controlId)
+    ?.removes?.find((object) => object["by-class"] === controlPartId);
+  const removeByItemNames = props.modificationDisplay.props.modificationAlters
+    ?.find((item) => item["control-id"] === controlId)
+    ?.removes?.find((object) => object["by-item-name"] === controlPartId);
+
+  if (
+    (removeByIds !== null && removeByIds !== undefined) ||
+    (removeByNames !== null && removeByNames !== undefined) ||
+    (removeByNS !== null && removeByNS !== undefined) ||
+    (removeByClass !== null && removeByClass !== undefined) ||
+    (removeByItemNames !== null && removeByItemNames !== undefined)
+  ) {
```

Comment:
```suggestion
  if (removeByIds || removeByNames || removeByNS || removeByClass || removeByItemNames) {
```

`null` and `undefined` are both falsey, so we can shorten this.

Reply:
Thanks @kylelaker for this suggestion. I will commit your suggestion.